data/mods/ssb/moves.ts

```diff
@@ -834,6 +834,48 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		type: "Ghost",
 	},
 
+	// PiraTe Princess
+	dungeonsdragons: {
+		accuracy: true,
+		basePower: 0,
+		category: "Status",
+		desc: "Prevents the target from switching out, adds Dragon to the target's type. Has a 5% chance to either confuse the user or guarentee the next attack to be a critical hit, 15% chance to raise users Attack, Defense, Special Attack, Special Defense or Speed by 1, and a 15% chance to raise both user's Special Attack and Speed by 1 stage.",
+		shortDesc: "Prevents the target from switching out, adds Dragon to the target's type. Randomly does more things. Please refer to docs.",
```

Comment:
```suggestion
		desc: "Prevents the target from switching out and adds Dragon to the target's type. Has a 5% chance to either confuse the user or guarantee that the next attack is a critical hit, 15% chance to raise the user's Attack, Defense, Special Attack, Special Defense, or Speed by 1 stage, and a 15% chance to raise user's Special Attack and Speed by 1 stage.",
		shortDesc: "Target: can't switch,+Dragon. Does other things.",
```
It's called shortDesc for a reason.